tests/integration/parity/common.py

```diff
@@ -181,7 +176,17 @@ def test_invalid_eth_signTypedData(self,
 
 
 class ParityTraceModuleTest(ParityTraceModuleTest):
-    pass
+    def test_trace_block(self, web3):
+        pytest.xfail("TODO: tracing not working on v2.5.13")
```

Comment:
In general, we prefer using the `@pytest.mark.xfail` decorator instead of using `pytest.xfail`, since pytest.xfail doesn't run anything else in the tests unless there is a reason for not running the rest of the test. Would it be possible to use the decorator here?

Here's some [documentation](https://docs.pytest.org/en/latest/skipping.html#xfail-mark-test-functions-as-expected-to-fail)